### ISSUE TITLE
--table-borders should be removed?

### ISSUE THREAD

**Issue by miekg** [ietf-tools/xml2rfc#722]
--table-borders changes the way a document looks and should be removed or moved into the <rfc> tag.

I *think* most folks are/were in favor of just removing it.

**Comment by cabo** [comment]
> I _think_ most folks are/were in favor of just removing it.

The command line option is somewhat nonsensical, indee.

Not at all the formatting parameter.
Most usefully as an attribute.
I wouldn't mind a PI either.

**Comment by miekg** [comment]
attribute is a nice idea.

Also in similar vein: `--table-hyphen-breaks` should be made a default and not a flag?

**Comment by cabo** [comment]
These are what we call "tweaks" in the world of document formatting.
It does not make sense to apply tweaks globally for a document, unless that is so uniform that all (e.g.) tables are the same.  Instead, the tweaks should be with the specific item.  PIs were meant to be one channel for tweaks (they can also be used to indicate whether the RFC should be stapled after printing, but that is not something we embrace).  We do have several tweaks that are attributes, and nothing is wrong adding a few more, if we don't go overboard.

**Comment by cabo** [comment]
I see a lot of people using ASCII artwork because they don't get the table rendering they need.
That of course can impair the HTML/PDF rendering while improving the TXT rendering.

**Comment by dkg** [comment]
It would be great to have the table borders in the text output rendered with Unicode U+2500 BOX DRAWING characters instead of ASCII art, and i agree that it doesn't make a lot of sense as a command-line option.

The directive should be able to be placed either be at the document level, or as an option for specific tables to override the document-level preference.

**Comment by jrlevine** [comment]
This gets into the swamp of what non-ASCII characters we allow in the text rendering.
I agree that the RFC 7997 turned out to be too strict but I really do not want to undo this one casual hack at a time.

In the meantime, those HTML tables look pretty nice.

**Comment by miekg** [comment]
> In the meantime, those HTML tables look pretty nice.

yep. But call me old or whatever, but I'm still very charmed by the xml2rfc text output.

**Comment by cabo** [comment]
So maybe  we should starting labeling issues like this post-7997 (i.e., revisit when we have managed to enter a state where we can discuss 7997).

**Comment by miekg** [comment]
sgtm - none of this seems urgent
